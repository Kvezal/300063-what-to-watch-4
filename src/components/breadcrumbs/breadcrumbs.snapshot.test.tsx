import * as React from "react";
import * as render from "react-test-renderer";
import {MemoryRouter} from "react-router-dom";

import Breadcrumbs from "./breadcrumbs";


const breadcrumbs = [
  {name: `name-1`, href: `href-1`},
  {name: `name-2`, href: `href-2`},
  {name: `name-3`},
];

test(`should render component`, () => {
  const tree = render.create(
      <MemoryRouter>
        <Breadcrumbs
          list={breadcrumbs}
        />
      </MemoryRouter>
  )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
