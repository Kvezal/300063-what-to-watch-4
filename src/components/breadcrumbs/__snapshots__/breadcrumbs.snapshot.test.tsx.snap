// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<nav
  className="breadcrumbs"
>
  <ul
    className="breadcrumbs__list"
  >
    <li
      className="breadcrumbs__item"
    >
      <a
        aria-current={null}
        className="breadcrumbs__link"
        href="/href-1"
        onClick={[Function]}
      >
        name-1
      </a>
    </li>
    <li
      className="breadcrumbs__item"
    >
      <a
        aria-current={null}
        className="breadcrumbs__link"
        href="/href-2"
        onClick={[Function]}
      >
        name-2
      </a>
    </li>
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
      >
        name-3
      </a>
    </li>
  </ul>
</nav>
`;
